### PR TITLE
WIP: Test snapshot CRDs and controller/sidecar images

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR updates snapshot CRDs and controller/sidecar images

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
```docs

```

### CODE REVIEW

File: cluster/addons/volumesnapshots/volume-snapshot-controller/volume-snapshot-controller-deployment.yaml [hunk 1]
```diff
@@ -22,6 +22,6 @@ spec:
       serviceAccount: volume-snapshot-controller
       containers:
         - name: volume-snapshot-controller
-          image: k8s.gcr.io/sig-storage/snapshot-controller:v3.0.2
+          image: docker.io/xyang105/snapshot-controller:apiv1 #k8s.gcr.io/sig-storage/snapshot-controller:v3.0.0
```
Comment (jeffvance) [inline cluster/addons/volumesnapshots/volume-snapshot-controller/volume-snapshot-controller-deployment.yaml:25]: is this a permanent change?

File: test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ spec:
       serviceAccountName: csi-gce-pd-controller-sa
       containers:
         - name: csi-snapshotter
-          image: k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.2
+          image: docker.io/xyang105/csi-snapshotter:apiv1 #gcr.io/gke-release/csi-snapshotter:v2.1.1-gke.0
```
Comment (jeffvance) [inline test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml:24]: ditto

File: test/utils/image/manifest.go [hunk 3]
```diff
@@ -69,6 +70,7 @@ func initReg() RegistryList {
 		GcAuthenticatedRegistry: "gcr.io/authenticated-image-pulling",
 		DockerLibraryRegistry:   "docker.io/library",
 		DockerGluster:           "docker.io/gluster",
+		DockerTestSnapshot:      "docker.io/xyang105",
```
Comment (jeffvance) [inline test/utils/image/manifest.go:73]: ditto

File: cluster/addons/volumesnapshots/volume-snapshot-controller/volume-snapshot-controller-deployment.yaml (hunk 1 above)
Comment (xing-yang) [reply cluster/addons/volumesnapshots/volume-snapshot-controller/volume-snapshot-controller-deployment.yaml:25]: No.  I'm testing the images.  It's not released yet so I have to use a temporary image.
Let me mark the PR WIP.

**Comment by xing-yang** [comment]
Hi @msau42 ,

This PR includes the v1 snapshot CRDs and snapshot controller, csi-snapshotter sidecar, and csi-provisioner sidecar using the v1 CRDs, but without test/e2e changes.  It has passed CI.

**Comment by msau42** [comment]
Awesome thanks! This shows that existing v1beta users will still function with an upgraded controller and driver.

**Pull request closed without merging**